packages/nuxt-windicss/src/module.ts

```diff
@@ -89,8 +72,8 @@ const defineNuxtWindiCSSModule = defineNuxtModule<NuxtWindiOptions>(nuxt => ({
         // avoid being too verbose
         if (nuxtWindiOptions.displayVersionInfo && nuxt.options.dev) {
           nuxt.hook('build:before', () => {
-            const { version } = requireModulePkg('windicss')
-            logger.info(`\`nuxt-windicss v${defineModuleMeta().version}\` using \`windicss v${version}\` running with config: \`${configType}\`.`)
+            const { version: windiVersion } = requireModulePkg('windicss')
```

Comment:
Note: This call is **unsafe** because search path of `requireModulePkg` does not includes `nuxt-windicss`. It works by depending on `node_modules` structure.

Reply:
Ah true, in that case I'll remove it as it's not needed